Ical.Net/DataTypes/PeriodList.cs

```diff
@@ -121,11 +124,11 @@ public void Insert(int index, Period item)
     {
         EnsureConsistentTimezoneAndPeriodKind(item);
         if (Periods.Contains(item)) return;
-        Periods.Insert(index, item);
+        Periods?.Insert(index, item);
```

Comment:
This `?` hast just been removed one commit earlier.

Reply:
Removed